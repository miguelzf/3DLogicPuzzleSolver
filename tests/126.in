- - - - -
- P L B Y
- - - - -
- - - - -
- - - - -

- - - - Y
- - - - -
- - - - -
- - - - -
- - - - -

R - - - -
- - - - B
- - - - L
- - - - P
- - - - R
