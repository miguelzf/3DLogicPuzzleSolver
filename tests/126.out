
Solution:
r  r  r  r  r  
r  P  L  B  Y  
r  p  l  b  y  
r  p  l  b  y  
r  p  l  b  y  
=============
r  p  l  b  Y  ||  R  r  r  r  r  
r  p  l  b  b  ||  b  b  b  b  B  
r  p  l  l  l  ||  l  l  l  l  L  
r  p  p  p  p  ||  p  p  p  p  P  
r  r  r  r  r  ||  r  r  r  r  R  
